other/pascal_triangle.py

```diff
@@ -0,0 +1,69 @@
+"""
+wissamfawaz12@gmail.com | github.com/wissamfawaz
+This implementation demonstrates how to generate the
+elements of a Pascal's triangle.
+What is Pascal's triangle?
```

Comment:
Add a small description of the algorithm here.

Reply:
Done.